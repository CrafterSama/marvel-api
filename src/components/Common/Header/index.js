import React from 'react'
import { HeaderContainer } from '../../../styles'
import Logo from './components/Logo'
import SearchInput from './components/SearchInput'
import Favorite from './components/Favorite'
import ChangeTheme from './components/ChangeTheme'

const Header = () => {

  return (
    <HeaderContainer>
      <Logo />
      <SearchInput />
      <Favorite />
      <ChangeTheme />
    </HeaderContainer>
  )
}

export default Header
