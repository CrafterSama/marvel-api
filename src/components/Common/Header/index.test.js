import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';
import Header from './index';

configure({ adapter: new Adapter() });

describe('Test for <Header /> Component', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
