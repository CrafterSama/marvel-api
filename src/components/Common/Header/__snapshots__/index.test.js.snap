// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <Header /> Component matches snapshot 1`] = `
<styles__HeaderContainer
  darkmode=""
>
  <Logo />
  <SearchInput />
  <Favorite />
  <DarkMode />
</styles__HeaderContainer>
`;
